docs/installation/local.md

```diff
@@ -1,5 +1,24 @@
 # Installing Loki Locally
 
+# RPM packages for Linux distros (openSUSE Linux)
+
+If you use openSUSE Linux and RPM based distros, you can install packaged version of Loki with:
+
+1) Add the repository ``https://build.opensuse.org/package/show/server:monitoring/loki` to your system
```

Comment:
Could you add details on how users can add the repository?

Reply:
ok